scripts/variantstore/wdl/GvsWithdrawSamples.wdl

```diff
@@ -0,0 +1,97 @@
+version 1.0
+
+workflow GvsWithdrawSamples {
+
+  input {
+    String dataset_name
+    String project_id
+
+    Array[String] sample_names
+
+    File? withdraw_samples_gatk_override
+    String? service_account_json_path
+  }
+
+  String sample_info_table = "sample_info"
+
+  call WithdrawSamples {
+    input:
+      project_id = project_id,
+      dataset_name = dataset_name,
+      sample_info_table = sample_info_table,
+      sample_names = sample_names,
+      gatk_override = withdraw_samples_gatk_override,
+      service_account_json_path = service_account_json_path
+  }
+
+  output {
+    Int num_rows_updated = WithdrawSamples.num_rows_updated
+  }
+}
+
+task WithdrawSamples {
+  input {
+    String project_id
+    String dataset_name
+
+    String sample_info_table
+    Array[String] sample_names
+
+    # runtime
+    File? gatk_override
+    String? service_account_json_path
+  }
+
+  String has_service_account_file = if (defined(service_account_json_path)) then 'true' else 'false'
+
+  meta {
+    description: "Withdraw Samples from GVS by marking them as 'withdrawn' in the sample_info table"
+    volatile: true
+  }
+
+  command <<<
+    set -e
+    set -x
+
+    # make sure that sample names were actually passed, warn and exit if empty
+    num_samples=~{length(sample_names)}
+    if [ $num_samples -eq 0 ]; then
+      echo "No sample names passed. Exiting"
+      exit 0
+    fi
+
+    export GATK_LOCAL_JAR=~{default="/root/gatk.jar" gatk_override}
```

Comment:
It seems `gatk_override` isn't actually needed in this WDL and could be removed (there are no GATK invocations here).

Reply:
Thanks.